cps_data/pycps/taxunit.py

```diff
@@ -0,0 +1,195 @@
+INCOME_TUPLES = [
```

Comment:
Philosophical question about this file: Why make a new `TaxUnit` object, rather than a DataFrame?

Reply:
This was mostly to make it easier for me to conceptualize what was happening. Treating the tax unit as its own object made it easier for me to form the logic of creating a tax unit, without severely complicating transforming that `TaxUnit` object into a row in the final DataFrame.